docs/reference.md

```diff
@@ -95,6 +101,29 @@ response.json = {'some': 'dict'}
 response.json = [{'some': 'dict'}, {'another': 'one'}]
 ```
 
+### Multipart
+
+Responsible of the parsing of `request.body`.
+
+#### Methods
+
+- **parse(content_type: str)**: returns a tuple
+  ([Form](#form) instance, [Files](#files) instance) filled with data
+  from `feed_data`
+- **feed_data(data: bytes)**: incrementally fills [Form](#form) and
+  [Files](#files) objects with bytes from the body
+
+
+### Form
+
+Allow to access casted POST parameters from `request.body`.
+
+
+### Files
+
+Allow to access POSTed files from `request.body`.
```

Comment:
We should document `get` and `list` method (from Multidict).

Reply:
Done.